### ISSUE TITLE
Create expense list layout

### ISSUE THREAD

**Issue by TheLastBanana** [CMPUT301W15T07/TravelTracker#53]
Refer to the UI diagram:
![UI diagram](https://raw.githubusercontent.com/CMPUT301W15T07/TravelTracker/master/doc/UI_diagram.png)

**Comment by kuzi117** [comment]
Should this be ExpenseList or ItemList.

**Comment by TheLastBanana** [comment]
Good point. For clarity's sake, it should probably be ItemList.

**Comment by kuzi117** [comment]
All right, I'll change the UI diagram as well.

**Comment by kuzi117** [comment]
It's done, though the list item view will undoubtedly need tweaking once we can fill with data.
(1c0db83)

**Comment by therabidsquirel** [comment]
It was originally "Items List" and "Item Info", but I changed it when the TA said this in feedback:

>I suggest to use Expense Items/Item to instead of "Items List" and "Item Info" in "Items List" and "Item Info" screen. It will be more clear to understand the page feature for the customer.

While "Item/Items" does follow our use in the glossary, I feel to a user of the app it's not entirely clear what that means. "Expense Item" is clear, but I felt the name was too long (Expense Items List and Expense Item Info), so I changed it back to "Expense" for the purpose of the UI. I feel "Expense Item" is perhaps a better term for use in the UI, as I really don't think "Item" is clear enough.
